virttest/qemu_devices/qcontainer.py

```diff
@@ -2246,6 +2246,8 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
         if backend in ['unix_socket', 'file', 'pipe', 'serial',
                        'tty', 'parallel', 'parport']:
             chardev_param.update({'path': file_name})
+            if (backend == 'pipe' and params.get('auto_create_pipe', 'no') == "yes"):
+                process.system("mkfifo %s" % file_name)
```

Comment:
@yanan-fu hi, don't we create pipe default? it means the default value of `auto_create_pipe` is` yes`.

Reply:
make sense,  @michiboo  please update, and also the tp-qemu cfg. thanks!